models/src/main/protobuf/DeployServiceV1.proto

```diff
@@ -184,3 +188,10 @@ message BondStatusResponse {
     bool isBonded   = 2;
   }
 }
+
+message StatusResponse {
+  oneof message {
+    ServiceError error  = 1;
```

Comment:
@tgrospic Should we use some new style of error message for grpc API?
Actually there is no internal error for the statusAPI. Maybe we can just skip this?

Reply:
Oh. I thought this api would just static information except peer count.

Reply:
@zsluedem The _error_ here is just to be consistent with other results. But yes, we should get rid of errors inside service definitions and use _protocol_ errors. E.g for gRPC https://www.grpc.io/docs/guides/error/.